pkg/client/dtclient/config_client_test.go

```diff
@@ -408,14 +408,12 @@ func Test_getObjectIdIfAlreadyExists(t *testing.T) {
 			defer server.Close()
 
 			dtclient, _ := NewDynatraceClientForTesting(server.URL, server.Client(), nil)
-			got, err := dtclient.getObjectIdIfAlreadyExists(context.TODO(), testApi, server.URL, tt.givenObjectName, nil)
+
+			_, _, err := dtclient.configExistsByName(context.TODO(), testApi, tt.givenObjectName)
 			if (err != nil) != tt.wantErr {
 				t.Errorf("getObjectIdIfAlreadyExists() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
-			if got != tt.wantFoundId {
```

Comment:
why did you remove this check? If it is really not needed please remove the `wantFoundId` field from the testcases =

Reply:
accidentally